domains/x6rh3pe4jkng.exocubeyt.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ExoCubeYT",
+        "email": "umanbhat7780@gmail.com"
+    },
+    "record": {
+        "CNAME": "gv-rynyaxcxi6w6lk.dv.googlehosted.com"
```

Comment:
We don't support TXT verification

Reply:
But blogger needs it so what do I do, also blogger says this

We have not been able to verify your authority to this domain. On your domain registrar's website, locate your Domain Name System (DNS) settings and enter the following two CNAMEs: (Name: exocubeyt, Destination: ghs.google.com) and (Name: x6rh3pe4jkng, Destination: gv-rynyaxcxi6w6lk.dv.googlehosted.com). See https://support.google.com/blogger/answer/1233387 for detailed instructions. Support page link

Reply:
We can't allow verification records unfortunately, so you will need to use another service.